AWSutils.sh

```diff
@@ -117,6 +126,12 @@ function InstallCLIv1 {
       rm -rf "${CHROOTMNT}/tmp/awscli-bundle.zip" \
          "${CHROOTMNT}/tmp/awscli-bundle" || \
         err_exit "Failed cleaning up install files"
+   elif [[ ${CLIV1SOURCE} == pip,* ]]
+   then
+      # Make sure Python3 is present
+      EnsurePy3
+
+      python3 -m pip install --upgrade "${CLIV1SOURCE/pip*,}"
```

Comment:
Does this need to run in the chroot? Also, recommend removing the upgrade flag...

And, there could be some trickiness to supporting this approach when there is no connectivity to the Internet-hosted pypi.org, but can deal with that when we come to it.

Reply:
D'oh! Fixing...